.circleci/config.yml

```diff
@@ -36,16 +36,21 @@ jobs:
       - checkout
       - run: npm ci
       - run: npx semantic-release
-      - run: sed -i 's/@percy\/agent/percy/g' package.json
-      - run: npm ci
-      - run: npx semantic-release
+      - run: |
+          sed -i 's/@percy\/agent/percy/g' package.json
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+          npm publish
```

Comment:
Will this fail CI when it tries to publish over the same version?

> What this means is that npm publish -f will not work. The first step, where it deletes the old copy, will work. But then trying to publish over it will fail with a 403 Forbidden response. You’ll have to change the version number to make it work.

https://blog.npmjs.org/post/77758351673/no-more-npm-publish-f

Reply:
You're absolutely right. I reran a build that was green and now it's red https://circleci.com/gh/percy/percy-agent/1970

I'll find a workaround for that.

Reply:
This is the little node script we used to release packages: https://github.com/bigtestjs/meta/blob/master/bin/release

Basically checks the package.json version in the npm registry and if it's not present, go ahead and release.

Reply:
@Robdel12 ready for another look